lib/utils/telemetry/generate-payload.js

```diff
@@ -258,8 +258,10 @@ module.exports = ({
     payload.isConfigValid = getConfigurationValidationResult(configuration);
     payload.dashboard.orgUid =
       serverless && serverless.isDashboardEnabled ? serverless.service.orgUid : undefined;
-    payload.console.orgUid =
-      serverless && serverless.console.isEnabled ? serverless.service.orgUid : undefined;
+    if (_.get(serverless, 'console.isEnabled')) {
+      payload.console.orgUid = serverless.service.orgUid;
+      payload.console.extensionVersion = serverless.console._usedExtensionLayerVersionPostfix;
```

Comment:
So in the end we will be reporting the extension version, right? Are there additional changes needed on the backend side for that?

Reply:
Just noticed that, thanks for clarifying 👍